### PR TITLE
Improve and add new client apps

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

**Comment by danielpassos** [comment]
@abstractj @corinnekrych @cvasilak Could you give it a spin?

File: Shoot/README.md [hunk 1]
```diff
@@ -40,58 +42,58 @@ To test Cordova client go to [TODO]().
...
+    "publicClient" : true
+} ]
 ```
 
-* Define your services to list, upload, get images. Check ```src/main/java/org/jboss/aerogear/shoot/PhotoService.java```
+Define your services to list, upload, get images. (Check `src/main/java/org/jboss/aerogear/shoot/PhotoService.java`)
 
-* To secure your endpoints, user wildfly keyclaok adapter. This simple demo uses a "per war approach". Refer to [Keycloak adapters chapter for more details](http://docs.jboss.org/keycloak/docs/1.0-final/userguide/html/ch07.html)
-Go in ```src/main/webapp/WEB-INF/web.xml```
+To secure your endpoints, user wildfly Keyclaok adapter. This simple demo uses a "per war approach". Refer to [Keycloak adapters chapter for more details](http://docs.jboss.org/keycloak/docs/1.0-final/userguide/html/ch07.html)
```
Comment (corinnekrych) [inline Shoot/README.md]: /Keyclaok/Keycloak/

File: Shoot/README.md [hunk 2]
```diff
@@ -1,38 +1,40 @@
 ## Shoot'nShare backend
-A simple example to demo OAuth2 authorization code grant with Keycloak.
 
+A simple example to demo OAuth2 authorization code grant with Keycloak.
 
 ### Pre-requisites
 
-* use [Keycloak Appliance Distribution (version 1.0.4.Final)](http://docs.jboss.org/keycloak/docs/1.0.4.Final/userguide/html/server-installation.html#Appliance_install) comes with a preconfigured Keycloak server (based on Wildfly). 
+Shoot'nShare backend uses [Keycloak](http://keycloak.jboss.org/) to authenticate or client apps, so we need deploy and configure the [Keycloak](http://keycloak.jboss.org/)
```
Comment (corinnekrych) [inline Shoot/README.md]: /to authenticate or client apps,/to authenticate our client apps,/
/so we need deploy and configure the [Keycloak](http://keycloak.jboss.org/)/so we need to deploy and configure the [Keycloak](http://keycloak.jboss.org/) server./

File: Shoot/README.md [hunk 3]
```diff
@@ -40,58 +42,58 @@ To test Cordova client go to [TODO]().
...
+
+<security-role>
+    <role-name>user</role-name>
+</security-role>
 ```
 
 NOTE: For simplicity our demo does not use https BUT all OAuth2 protected REST points should be using SSL. If you are using WildFly or EAP follow this [link to enable HTTPS](https://docs.jboss.org/author/pages/viewpage.action?pageId=66322705) or consult your application server's documentation page.
 
-and use ```src/main/webapp/WEB-INF/keycloak.json```, you link your java endpoint with keyclaok realm application. Here the application is names ```shoot-services```
+and use `src/main/webapp/WEB-INF/keycloak.json`, you link your java endpoint with keyclaok realm application. Here the application is names `shoot-services`
```
Comment (corinnekrych) [inline Shoot/README.md]: /keyclaok/keycloak/

**Comment by corinnekrych** [comment]
Some typos to correct - Keyclaok one coming from me :(
+1 to merge when fixed

**Comment by danielpassos** [comment]
@corinnekrych Done. Thanks for review that.

**Pull request merged**